### PR TITLE
Add additional prop type to tooltip caption

### PR DESCRIPTION
**Pull request by mxmilian**

### CODE REVIEW

File: src/components/tooltip/Tooltip.tsx [hunk 1]
```diff
@@ -3,7 +3,7 @@ import { StyledTooltip, TooltipWrapper } from "./styles";
 import { useTooltipHover } from "./hooks";
 
 export type Props = {
-  caption: string;
+  caption: string | React.ReactNode;
```
Comment (nowyDEV) [inline src/components/tooltip/Tooltip.tsx:6]: ReactNode includes string

```suggestion
  caption: React.ReactNode;
```

**Review by nowyDEV — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -16308,13 +16308,6 @@
   dependencies:
     "loose-envify" "^1.0.0"
 
-"watchpack-chokidar2@^2.0.0":
```
Comment (raczyk) [inline yarn.lock:16311]: unnecessary change, revert this please

**Review by raczyk — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@nordcloud/gnui",
   "description": "Nordcloud Design System - a collection of reusable React components used in Nordcloud's SaaS products",
-  "version": "2.0.1",
+  "version": "2.0.2",
```
Comment (nowyDEV) [inline package.json:4]: I think we are adding a new functionality with this change and not fixing a bug, take a look how versioning works
https://semver.org/

**Review by nowyDEV — APPROVED** [review]
